playbooks/sqalab/run.yaml

```diff
@@ -0,0 +1,25 @@
+- hosts: all
+  tasks:
+    - name: 'scedule test run'
+      command: |
+        /snap/bin/weebl-tools.sqalab build add \
+          --deployment-branch {{ sqalab.branch }} \
+          --deployment-repo {{ sqalab.repo }} \
+          --lab {{ sqalab.lab }} \
+          --format json \
+      register: build_info
+    - name: 'wait for test run'
+      command: /snap/bin/weebl-tools.sqalab build show --format json {{ (build_info.stdout | from_json)[0].uuid }}
+      register: build_info
+      until: (build_info.stdout | from_json)[0].status in ["Finished", "Canceled", "Aborted"]
+      delay: 600
+      retries: 90
```

Comment:
So it this 600 minutes or 600 seconds? if the later, it's probably not long enough.  How does the 90 retries relate to the 600?

Reply:
So it checks every 600s if the test run is done with a maximum of 90 checks, so the timeout is 15h. The timeout from the SQA side is 12 h (it never takes that long though) + some time to wait for hardware to become available.

Reply:
We don't strictly need to wait for the test run to finish because the links printed below will exist before the test run is done. This is just to reflect the progress in zuul. Do you think that is reason enough to keep the container running?

Reply:
Okay, this sounds good. I think it's reasonable to keep it going; it's not going to be doing much and is just one vm.